{
  "comments": [
    {
      "key": {
        "uuid": "d58004d3_3e2e3e52",
        "filename": "test/lvol/test_cases.py",
        "patchSetId": 37
      },
      "lineNbr": 349,
      "author": {
        "id": 1011256
      },
      "writtenOn": "2017-09-26T12:58:04Z",
      "side": 1,
      "message": "everywhere else you return fail_count. Why can\u0027t you increment fail_count just return it?",
      "range": {
        "startLine": 349,
        "startChar": 23,
        "endLine": 349,
        "endChar": 31
      },
      "revId": "2f1e8b6250ce1dc116f1990c0ba6e4015cf0c538",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "cd653526_401e7e94",
        "filename": "test/lvol/test_cases.py",
        "patchSetId": 37
      },
      "lineNbr": 355,
      "author": {
        "id": 1011256
      },
      "writtenOn": "2017-09-26T12:58:04Z",
      "side": 1,
      "message": "tabs/spaces?",
      "range": {
        "startLine": 355,
        "startChar": 0,
        "endLine": 355,
        "endChar": 1
      },
      "revId": "2f1e8b6250ce1dc116f1990c0ba6e4015cf0c538",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "e1f0d581_fee1c820",
        "filename": "test/lvol/test_cases.py",
        "patchSetId": 37
      },
      "lineNbr": 358,
      "author": {
        "id": 1011256
      },
      "writtenOn": "2017-09-26T12:58:04Z",
      "side": 1,
      "message": "test number",
      "range": {
        "startLine": 358,
        "startChar": 15,
        "endLine": 358,
        "endChar": 17
      },
      "revId": "2f1e8b6250ce1dc116f1990c0ba6e4015cf0c538",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": false
    }
  ]
}